server/php-engine.h

```diff
@@ -61,6 +61,8 @@ class PhpScript;
 
 extern std::optional<PhpScript> php_script;
 
+constexpr double epsilon = 1e-9;
```

Comment:
may be in upper case?

Reply:
Remove constexpr epsilon and add optional of double